websites/W/Wolfery/presence.ts

```diff
@@ -0,0 +1,34 @@
+const presence = new Presence({
+		clientId: "1244143703660953651",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://forum.wolfery.com/uploads/default/original/1X/96ef8d56ab7b19e7f69cf42fc9109c98bf93e588.png",
+}
+
+// TODO: Add LFRP Flag (v1.1.0)
+// TODO: Find potential links for adding presence button to open character sheet. (v1.1.0)
+
+const detailMsg = "Roleplaying on Wolfery.com";
+let stateMsg = "";
+
+presence.on("UpdateData", async () => {
+	const [characterPrivacy] = await Promise.all([
+		presence.getSetting<boolean>("characterPrivacy"),
+	]);
+
+	let characterName = document.querySelector(".namesection--title").textContent;
+
+	if (characterPrivacy) {
+		stateMsg = "";
+		characterName = "";
+	} else stateMsg = `Playing As: ${characterName}`;
+
+	presence.setActivity({
+		details: detailMsg,
+		state: stateMsg,
+		largeImageKey: Assets.Logo,
+		startTimestamp: browsingTimestamp,
+	});
+});
```

Comment:
Are you able to show any more information, or add other subpages?

Reply:
The website is a single page application and doesn't have much in terms of information. So far, in my current branch, it will show the characters name on the website of the user signed in. I need to add a check if they're signed in, which I will add here in a few.

Reply:
I was able to add a few more information based on the header of the page that they were on. Namely just if they're "online" (aka awake) with a character, choosing a character or at the edit screen.